tests/external_botocore/conftest.py

```diff
@@ -66,6 +66,10 @@
 )
 
 
+def llm_token_count_callback_success(model, content):
```

Comment:
I think we can drop the `_success` since there's only one of these now:
```suggestion
def llm_token_count_callback(model, content):
```